vmm/src/api/openapi/cloud-hypervisor.yaml

```diff
@@ -467,6 +467,8 @@ components:
         hugepages:
           type: boolean
           default: false
+        host_numa_node:
+          type: uint64
```

Comment:
Linux only supports a maximum of 1024 NUMA nodes (by default) and given the ACPI spec uses a u32 for this I think this should also be a u32 here.

Reply:
Fixed by https://github.com/cloud-hypervisor/cloud-hypervisor/pull/1658